Lib/profile.py

```diff
@@ -387,8 +387,9 @@ def simulate_cmd_complete(self):
 
     def print_stats(self, sort=-1):
         import pstats
-        pstats.Stats(self).strip_dirs().sort_stats(sort). \
-                  print_stats()
+        if not hasattr(sort, "__iter__") or isinstance(sort, str):
```

Comment:
It can only be `int` or `str` or `SortKey`. So  it can be written as
```suggestion
        if isinstance(sort, (str, int, pstats.SortKey)):
```
(you can even exclude `SortKey`, because it is a `str` subclass, but this is an implementation detail)

On other hand, you only documented tuple as accepted type. Passing dict, or set, or bytes, or iterator would be an error. deque is too exotic for this. So, it may be simpler and more safe to write this as:
```suggestion
        if not isinstance(sort, tuple):
```